docs/source/modules/held_suarez.rst

```diff
@@ -0,0 +1,402 @@
+:smaxis: Semi-major axis of planet's orbit. Default :math:`1.5\times10^{6}\,\text{m}`. **Note**, this value seems to low. Upon further inspection of the code, it seems it is not used anywhere. It should be used to compute a seasonal cycle due to eccentricity, however this functionality does not seem to be implemented at the moment. 
+:orbital_period: Period of planet's orbit. Seems to override ``orbital_period`` set in ``constants_nml``. Default :math:`360\,\text{days}`. 
+:heat_capacity: Normalised surface heat capacity :math:`C_{\text{g}}/d` (equivalent to mixed layer depth of :math:`1\,\text{m}`). Default :math:`4.2\times10^{6}\,\text{J}\,\text{K}^{-1}\,\text{m}^{-2}\,\text{m}^{-1}`.
+:ml_depth: Mixed layer depth :math:`d` (see above). Default :math:`1\,\text{m}`. 
+:spinup_time: Days to spin up surface temperature (decoupled from atmosphere) during initialisation. Default :math:`10800\,\text{days}`. Note: must be a multiple of orbital period. 
+:stratosphere_t_option: String determining temperature structure in the stratosphere. Possible options include ``'c_above_tp'``, ``'hs_like'``, ``'extend_tp'`` (see above for description). If any other option is specified, then the atmosphere is 'all troposphere', and the temperature will decrease with decreasing pressure until it reaches zero. 
+:t_strat: Isothermal stratosphere temperature, :math:`T_{\text{strat}}`. Defualt :math:`200\,\text{K}`. Used when ``stratosphere_t_option`` is ``'c_above_tp'`` or ``'hs_like'``. This namelist variable is the same as that used by the regular Newtonian damping. 
+
+**Local heating**
+
+:local_heating_option: String indicating the option used for local heating. Valid options are ``'from_file'`` and ``'Isidoro'`` (see documentation for meaning), otherwise local heating is not performed. Default ``''``. 
+
+:local_heating_file: Name of input netCDF file for :math:`Q_{\text{local}}` wind used if ``local_heating_option = 'from_file'``. File name should exclude ``.nc`` suffix. 
+
+The following options are used if ``local_heating_option = 'Isidoro'``
+
+:local_heating_srfamp: Surface amplitude of local heating :math:`A`. Default :math:`0.0\,\text{K}\,\text{day}^{-1}`. 
+:local_heating_xwidth: Longitudinal width of local heating :math:`\Delta\lambda`. Default :math:`10^{\circ}`. 
+:local_heating_ywidth: Latitudinal width of local heating :math:`\Delta\theta`. Default :math:`10^{\circ}`. 
+:local_heating_xcenter: Longitudinal center of heating :math:`\lambda_{0}`. Default :math:`180^{\circ}`. 
+:local_heating_ycenter: Longitudinal center of heating :math:`\lambda_{0}`. Default :math:`45^{\circ}`. 
+:local_heating_vert_decay: Vertical decay of local heating in pressure :math:`\Delta p`. Default :math:`10^{4}\,\text{Pa}`.  
+
+
+
+**Other**
+
+There are also optins for a tracer source and sink, ``trflux`` and ``trsink``. These are currently undocumented, however, as Isca only uses the ``hs_forcing`` when the model is dry and there are no tracers. 
```

Comment:
Perhaps change `there are no tracers` to `there are no active tracers`, since it is possible to add passive tracers in the dry model.